Packs/Core/Playbooks/playbook-Netcat_Makes_or_Get_Connections.yml

```diff
@@ -0,0 +1,955 @@
+          "y": 715
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: 55b7d40a-0f6c-4ed8-8689-c7ec3c842dbd
+    type: regular
+    task:
+      id: 55b7d40a-0f6c-4ed8-8689-c7ec3c842dbd
+      version: -1
+      name: Get Commandline prevalence
+      description: Get the prevalence of a process_command_line, identified by process_command_line.
+      script: '|||core-get-cmd-analytics-prevalence'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "30"
+    scriptarguments:
+      process_command_line:
+        simple: ${alert.osparentcmd}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1075
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "23":
+    id: "23"
+    taskid: c6c19d37-04a8-4d09-8bca-3ef43b270e5f
+    type: title
+    task:
+      id: c6c19d37-04a8-4d09-8bca-3ef43b270e5f
+      version: -1
+      name: Done
+      description: commands.local.cmd.close.inv
+      type: title
+      iscommand: false
+      brand: Builtin
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 2450
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "24":
+    id: "24"
+    taskid: bfbce647-7068-4dbc-8450-d3b7436aeca4
+    type: title
+    task:
+      id: bfbce647-7068-4dbc-8450-d3b7436aeca4
+      version: -1
+      name: Investigation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "39"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1450
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "25":
+    id: "25"
+    taskid: 939d4df1-3b9d-4a02-8634-59f622bffba6
+    type: regular
+    task:
+      id: 939d4df1-3b9d-4a02-8634-59f622bffba6
+      version: -1
+      name: Check Previous similar Alerts
+      description: |-
+        Finds past similar alerts based on alert fields' similarity. Includes an option to also display indicators similarity.
+        Note: For the similarity calculation, at least one field must be provided in one of the "similarTextField", "similarCategoricalField", or "similarJsonField" arguments.
+      scriptName: DBotFindSimilarAlerts
```

Comment:
Did you check it? what are the similarity fields it's relying on?
Shouldn't it be better to use SearchAlertsV2 with a defined query?